hana/lib/HANAService.js

```diff
@@ -528,6 +530,7 @@ class HANAService extends SQLService {
     }
 
     INSERT_entries(q) {
+      this.values = undefined
```

Comment:
In which situations do we need to reset the values?

Reply:
By setting the `values` to `undefined` it informs the `val` function that it should just inline the values instead of pushing them onto the `values` array. When the `INSERT` statement is being rendered it is important to only have the `JSON` payload be the parameters. This also prevents `session_context` and other functions from being rendered with place holders while their values should be rendered into the sql directly instead.